OVRUI/src/Player/Player.js

```diff
@@ -371,6 +375,28 @@ export default class Player {
   }
 
   /**
+   * Handles taking ponterlock in response to pointer input being restricted
+   */
+  handlePointerRestricted() {
+    var pointerLockElement = this.glRenderer.domElement;
+    if (pointerLockElement && pointerLockElement.requestPointerLock) {
+      pointerLockElement.requestPointerLock();
+    }
+  }
+
+  /**
+   * Handles releasing ponterlock in response to pointer input being unrestricted
+   */
+  handlePointerUnrestricted() {
+    var currentPointerLockElement = document.pointerLockElement;
```

Comment:
Looks like flow's BOM reference is unaware of pointerLockElement. I'll send a PR, but in the meantime can you put a `// $FlowFixMe` comment above this line?

Fixing the error on 395 should be as simple as changing `document.exitPointerLock` to `typeof document.exitPointerLock === 'function'` below. Before sending an update, can you also make sure that running `yarn flow` produces an error-free check?